Makefile

```diff
@@ -7,7 +7,7 @@ CLIPPY_OPTS := -D warnings -D clippy::clone_on_ref_ptr -D clippy::enum_glob_use
 	-A clippy::mutable_key_type -A clippy::upper_case_acronyms
 CKB_TEST_ARGS := ${CKB_TEST_ARGS} -c 4
 INTEGRATION_RUST_LOG := info,ckb_test=debug,ckb_sync=debug,ckb_relay=debug,ckb_network=debug
-CARGO_TARGET_DIR ?= "../target"
+CARGO_TARGET_DIR ?= "target"
```

Comment:
The follow line could not work as expected.

https://github.com/nervosnetwork/ckb/blob/28272b0197151c20a27b0797f0e2510a7b685c4f/Makefile#L31

Reply:
please review again